starter-code/lib/harryPotter.js

```diff
@@ -8,3 +8,20 @@
                    "4 October", "Filius Flitwick", "17 October", "Molly Weasley",
                    "30 October", "Bill Weasley", "29 November", "Rubeus Hagrid",
                    "6 December", "Charlie Weasley", "12 December"];
+
+var moreBirthdays = ["Lily Evans", "30 January", "James Potter", "27 March",
+                    "Dudley Dursley", "30 June", "Tom Riddle", "31 December"];
+
+
+
+function birthdayArray() {
+  return _.chunk(birthdays,2);
+}
+console.log(birthdayArray());
+
+
+
+function allBirthdays() {
+  return _.concat(birthdayArray(), _.chunk(moreBirthdays,2));
```

Comment:
About the _.chunk(), if you need to do the same thing more than one time, better make a function.